hw/vfio/container.c

```diff
@@ -857,6 +857,16 @@ static bool vfio_device_get(VFIOGroup *group, const char *name,
 
 static void vfio_device_put(VFIODevice *vbasedev)
 {
+    if (vbasedev->regions != NULL) {
```

Comment:
this has already confused at least two people, think a rename to reginfo is needed.